avconv.js

```diff
@@ -47,27 +47,98 @@ function findTime(data) {
     return time;
 }
 
-function findVideoMetaData(data) {
-    var result = /Stream #([^:]+): Video: ([^,]+), ([^,]+), (\d+)x(\d+)/.exec(data),
-        meta;
-
-    if (result && result[1]) {
-        meta = {
-            video: {
-                track:     result[1],
-                codec:     result[2],
-                format:    result[3],
-                width:     parseInt(result[4]),
-                height:    parseInt(result[5])
-            }
-        };
+var
+    VIDEO_META = {CODEC: 0, FORMAT: 1, RESOLUTION: 2, BITRATE:3, FPS: 4},
+    AUDIO_META = {CODEC: 0, SAMPLERATE: 1, SPATIALIZATION: 2, ENCODING: 3, BITRATE: 4}
+;
+function parseMetaData(output) {
+    var
+        meta = {input:{},output:{}},
+        streamIndex,
+        streamData,
+        metaType,
+        ioData, tmp;
+
+    function mapInts(v) {return parseInt(v,10)}
```

Comment:
function `mapInts()` does not map anything itself, so change it to `formatInts()` or so.